### ISSUE TITLE
Problem installing: version does not satisfy requirement rich + no matching distribution found for rich[bug]: 

### ISSUE THREAD

**Issue by CRONOS090693** [invoke-ai/InvokeAI#6705]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

macOS

### GPU vendor

Apple Silicon (MPS)

### GPU model

M1 pro

### GPU VRAM

16GB

### Version number

v4.2.7

### Browser

chrome Version 126.0.6478.127

### What happened

Hello,
I first installed Python 3.11 on my mac, also Homebrew (and so also Xcode). I have a mac with M1 pro chip 2021 model. 
I followed the install instructions installing the latest version available: v4.2.7

When I copied the install.sh in my terminal and pressed enter I got the following response from the terminal. 
Does anybody know how I can resolve this? Any help would be greatly appreciated.

I would also highly appreciate you guys dumbing it down for me. I am not a developer and this is the first time ever I'm doing stuff in the terminal. I'm a creator excited about the possibilities this software has to offer and am just doing my best to gather the knowledge I need to get it up and running.

Thanks in advance
![Terminal error](https://github.com/user-attachments/assets/5040da26-95f2-4c8a-b31c-fe0cedc4d04b)

### What you expected to happen

For Stable Diffusion to be installed without any hickups since I downloaded and installed everything that is required and my computer meets all the requirements to be able to run it. 

### Additional context

I'm a total noob. please dumb it down or guide me to resources that I will be able to understand.

**Comment by psychedelicious** [comment]
It looks like python doesn't have its SSL config set up correctly.

If you installed python using the [official python installer](https://www.python.org/downloads/), you should have a `Python 3.11` folder under `Applications` in Finder. 

Inside that folder, look for `Install Certificates.command`. Double-click this to run a script to fix the SSL config. You may need to enter your password.

Then try the installer again.
